modules/sor/types.ts

```diff
@@ -1,6 +1,6 @@
 import { Chain } from '@prisma/client';
-import { GqlCowSwapApiResponse, GqlSorSwapType, GqlSorGetSwapsResponse, GqlSorSwapOptionsInput } from '../../schema';
-import { TokenAmount } from '@balancer/sdk';
+import { GqlSorSwapType, GqlSorGetSwapsResponse, GqlSorSwapOptionsInput } from '../../schema';
+import { TokenAmount } from './sorV2/sor-port/entities/tokenAmount';
```

Comment:
we could use this file to export any types we use internally to keep track of the module "pollution" into the other parts.